doc/whats-new.rst

```diff
@@ -52,6 +52,8 @@ Bug fixes
 
 - Fix the precision drop after indexing datetime64 arrays (:issue:`1932`).
   By `Keisuke Fujii <https://github.com/fujiisoup>`_.
+- Silenced irrelevant warnings issued by ``open_rasterio`` (:issue:`1964`).
```

Comment:
This isn't only rasterio warning anymore, is it?

Reply:
The other warnings only show up in our test suite, not to users. I'm not sure it's worth mentioning them.